dnscompress/label.go

```diff
@@ -1,10 +1,14 @@
-package dhcpv6
+package dnscompress
```

Comment:
what about calling this simply `rfc1035label`?

Reply:
Sounds good.